src/components/Home/index.ts

```diff
@@ -0,0 +1,3 @@
+import Home from './Home';
+
+export default Home;
```

Comment:
I don't see a need to have these components live in `<Component>.tsx` and then have a separate `index.ts` that just re-exports it. I'd just put the component code directly in `index.tsx`.